components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/resources/org/wso2/carbon/mediator/enrich/ui/i18n/JSResources.properties

```diff
@@ -2,3 +2,4 @@ specify.name=Specify the Name
 specify.value=Specify the Value
 specify.expression=Specify the Expression
 invalid.inline.xml=Invalid XML as Source-Inline content

```

Comment:
I suppose we can remove this property?

Reply:
It's used in other mediators like payload factory mediator